From 2ef6ccbb531fc1b0ffee9450417bd4feb3786a92 Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Fri, 25 Sep 2020 02:11:20 -0700
Subject: [PATCH 1677/1851] drm: xlnx: zynqmp_disp: Fix UNINIT type warning

commit c852af64ce66f3d1edf565351d5045705c532dd4 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix UNINIT type coverity warning.

Event var_decl: Declaring variable "ret" without initializer.
Event uninit_use: Using uninitialized value "ret".

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/zynqmp_disp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/xlnx/zynqmp_disp.c b/drivers/gpu/drm/xlnx/zynqmp_disp.c
index 057ce20f7b32..f27ad65dc20d 100644
--- a/drivers/gpu/drm/xlnx/zynqmp_disp.c
+++ b/drivers/gpu/drm/xlnx/zynqmp_disp.c
@@ -2091,7 +2091,7 @@ static int zynqmp_disp_layer_create(struct zynqmp_disp *disp)
 	unsigned int i;
 	int num_chans[ZYNQMP_DISP_NUM_LAYERS] = { 3, 1 };
 	const char * const dma_name[] = { "vid", "gfx" };
-	int ret;
+	int ret = -EINVAL;
 
 	for (i = 0; i < ZYNQMP_DISP_NUM_LAYERS; i++) {
 		char temp[16];
-- 
2.31.1

